tools/eksDistroBuildToolingOpsTools/cmd/eksDistroOpsProwPlugin/server.go

```diff
@@ -0,0 +1,269 @@
+			return fmt.Errorf("handle GolangPatchrelease: %w", err)
+		}
+	}
+	//TODO: add golangMinorMatches := golangMinorReleaseRe.FindAllStringSubmatch(ie.Issue.Title, -1)
+
+	return nil
+}
+
+func (s *Server) handle(logger *logrus.Entry, requestor string, issue *github.Issue, org, repo, title, body string, num int) error {
+	var lock *sync.Mutex
+	func() {
+		s.mapLock.Lock()
+		defer s.mapLock.Unlock()
+		if _, ok := s.lockMap[upstreamPickRequest{org, repo, num}]; !ok {
+			if s.lockMap == nil {
+				s.lockMap = map[upstreamPickRequest]*sync.Mutex{}
+			}
+			s.lockMap[upstreamPickRequest{org, repo, num}] = &sync.Mutex{}
+		}
+		lock = s.lockMap[upstreamPickRequest{org, repo, num}]
+	}()
+	lock.Lock()
+	defer lock.Unlock()
+
+	if err := s.HandleGolangPatchRelease(logger, issue); err != nil {
+		return fmt.Errorf("failed to handle Golang Patch Release: %w", err)
+	}
+
+	return nil
+}
+
+// Created based off plugins.FormatICResponse
+func FormatIEResponse(ie github.IssueEvent, s string) string {
+	return plugins.FormatResponseRaw(ie.Issue.Title, ie.Issue.HTMLURL, ie.Sender.Login, s)
+}
+
+func (s *Server) HandleGolangPatchRelease(l *logrus.Entry, upIss *github.Issue) error {
+	var golangVersionsRe = regexp.MustCompile(`(?m)(\d+.\d+.\d+)`)
+	var issNumRe = regexp.MustCompile(`(#\d+)`)
+	m := make(map[string]int)
+	for _, version := range golangVersionsRe.FindAllString(upIss.Title, -1) {
+		query := fmt.Sprintf("repo:%s/%s milestone:Go%s label:Security", constants.Golang, constants.Go, version)
+		milestoneIssues, err := s.ghc.FindIssuesWithOrg(constants.Golang, query, "", false)
+		if err != nil {
+			return fmt.Errorf("Find Golang Milestone: %v", err)
+		}
+		for _, i := range milestoneIssues {
+			for _, biMatch := range issNumRe.FindAllString(i.Body, -1) {
+				if m[biMatch] == 0 {
+					m[biMatch] = 1
+				}
+			}
+			return nil
+		}
+	}
+	for biNum := range m {
+		biInt, err := strconv.Atoi(biNum)
+		if err != nil {
+			return fmt.Errorf("Converting issue number to int: %w", err)
+		}
+		baseIssue, err := s.ghc.GetIssue(constants.Golang, constants.Go, biInt)
+		if err != nil {
+			return fmt.Errorf("Getting base issue(%s/%s#%d): %w", constants.Golang, constants.Go, biInt, err)
+		}
+		miNum, err := s.ghc.CreateIssue(constants.Aws, constants.EksdBuildTooling, baseIssue.Title, baseIssue.Body, 0, nil, nil)
+		if err != nil {
+			return fmt.Errorf("Creating mirrored issue: %w", err)
+		}
+		l.Info(fmt.Sprintf("Created Issue: %s/%s#%d", constants.Aws, constants.EksdBuildTooling, miNum))
+	}
+	return nil
+}
+
+func (s *Server) createComment(l *logrus.Entry, org, repo string, num int, comment *github.IssueComment, resp string) error {
+	if err := func() error {
+		if comment != nil {
+			return s.ghc.CreateComment(org, repo, num, plugins.FormatICResponse(*comment, resp))
+		}
+		return s.ghc.CreateComment(org, repo, num, fmt.Sprintf("In response to a upstreampick label: %s", resp))
```

Comment:
We'll want to change these references to `upstreampick` etc to reflect our current use-case; where and below in `createIssue`